gui/support/views.py

```diff
@@ -193,7 +193,7 @@ def ticket(request):
 
         license, reason = utils.get_license()
         if license:
-            company = license.customer_name
+            company = license.customer_name.decode()
         else:
```

Comment:
This (the support home.html template occurrence) also applies to the master branch as well, however in the master branch, this current views.py part is not even present. I suppose this is intentional and some other way was found in master to deal with/get to this information.